src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -510,6 +521,20 @@ public void DefaultExecuteCmdlet()
             }
         }
 
+        private VirtualMachineIdentity GetVMIdentityFromArgs()
```

Comment:
@praries880 same comment in this method about using `IsPresent` for checking usage of a `SwitchParameter` and potentially refactoring the code